packages/bot-web-ui/src/components/self-exclusion.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, ThemedScrollbars, Button } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { Formik, Form, Field } from 'formik';
+import { connect } from '../stores/connect';
+
+class SelfExclusion extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            self_exclusion_hide: true,
+            limit_error: '',
+        };
+    }
+    componentDidMount() {
+        if (!this.props.max_losses) {
+            this.setState({ self_exclusion_hide: false });
+        }
+    }
+    render() {
+        const {
+            max_losses,
+            set_max_losses,
+            initial_values,
+            updateSelfExclusion,
+            setLimitations,
+            onRunButtonClick,
+            run_limit,
+        } = this.props;
+        const self_exclusion_status = max_losses || 'Not set';
+
+        const onCancelSelfExclusion = async () => {
+            const { switchAccount, virtual_account_loginid, checkRestrictions } = this.props;
+            await switchAccount(virtual_account_loginid);
+            checkRestrictions();
+        };
+        const onUpdateLimitations = async () => {
+            if (!run_limit) {
+                this.setState({ limit_error: 'Run limit should set' });
+                return;
+            }
+            if (set_max_losses !== max_losses) {
+                const setLosses = await updateSelfExclusion({ max_losses: set_max_losses });
```

Comment:
We use snake_case for defining variables (that are not functions), i.e. `set_losses`.

Reply:
> We use snake_case for defining variables (that are not functions), i.e. `set_losses`.

Done